OpenOversight/tests/test_functional.py

```diff
@@ -73,41 +73,28 @@ def wait_for_element_to_be_visible(browser, locator, text, timeout=10):
         pytest.fail("Timed out while waiting for element to become visible")
 
 
+def scroll_to_element(browser, element):
+    browser.execute_script(
+        "arguments[0].scrollIntoView({ behavior: 'instant', block: 'center' });",
+        element,
+    )
+    return element
+
+
 def test_user_can_load_homepage_and_get_to_form(mockdata, browser, server_port):
     browser.get(f"http://localhost:{server_port}")
+    wait_for_element_to_be_visible(browser, By.ID, "cpd")
 
     # Complainant loads homepage
     assert "OpenOversight" in browser.title
-    with wait_for_page_load(browser):
-        browser.find_element_by_id("cpd").click()
+    element = browser.find_element(By.ID, "cpd")
+    scroll_to_element(browser, element)
+    element.click()
 
     page_text = browser.find_element_by_tag_name("body").text
     assert "Find an Officer" in page_text
 
 
-def test_user_can_use_form_to_get_to_browse(mockdata, browser, server_port):
```

Comment:
Can you elaborate why this test had to be removed? Was this functionality removed, or did the test no longer work with the updated framework?